web/styles/modal.css

```diff
@@ -547,18 +555,12 @@
 }
 
 .modal__body,
+.modal__content,
 .overlay-content {
-    .dropdown-widget-button,
-    .dropdown_widget_with_label_wrapper {
-        width: var(--modal-input-width);
-    }
-}
-
-.modal__body,
-.modal__content {
```

Comment:
This results in change of width of other dropdown widget buttons as well which we should avoid in this PR for now. One of them is in "Generate integration URL modal" for bots.